tool/simulate-dialogs.ts

```diff
@@ -169,11 +278,22 @@ export async function execute(args : any) {
         debug: false
     });
 
+    let parser = null;
+    if (args.nlu_server){
+        parser = ParserClient.get(args.nlu_server, args.locale);
+        await parser.start();
+    }
+
     await StreamUtils.waitFinish(
         readAllLines(args.input_file, '====')
         .pipe(new DialogueParser())
-        .pipe(new SimulatorStream(policy, simulator, schemas))
+        .pipe(new DialogueToPartialDialoguesStream())
+        .pipe(new SimulatorStream(policy, simulator, schemas, parser, tpClient, args.output_mistakes_only, args.locale))
         .pipe(new DialogueSerializer())
         .pipe(args.output)
     );
+
+    if (parser !== null) {
+        await parser.stop();
```

Comment:
No braces around single line if bodies, unless the other branch needs braces (eslint will complain)